src/element.js

```diff
@@ -0,0 +1,26 @@
+const utils = require('./utils');
+
+class Element {
```

Comment:
Can you or Gabor explain why we shifted from functional approach to the classical style? We could totally get rid of "this" and it would not be longer at all. This implementation is shorter just because the sendXXXCommand is now getting parameters not objects which we formatted with linebreaks. 

I understand that using class syntax makes it shiny and brief but I don't see why we should use the prototypal inheritence, since we don't want to dynamically link new functionalities for created sessions or elements. I know it's very fashioned nowadays, still don't think it is a good form of encapsulation and allows users to misuse the instances. 
For example: 
```
let button = new Element(session, 'XYZ');

await button.click();

button.elementId = 'ABC';

await button.click();
```